vendor/bat-native-ads/src/bat/ads/internal/ads/serving/permission_rules/unblinded_tokens_permission_rule.cc

```diff
@@ -32,6 +32,11 @@ std::string UnblindedTokensPermissionRule::GetLastMessage() const {
   return last_message_;
 }
 
+// static
+int UnblindedTokensPermissionRule::GetUnblindedTokensMinimumThreshold() {
+  return kUnblindedTokensMinimumThreshold;
+}
```

Comment:
Your thoughts on moving `constexpr int kUnblindedTokensMinimumThreshold = 10;` to a new file, such as `unblinded_tokens_permission_rule_constants.h`?